xapian-applications/omega/index_file.cc

```diff
@@ -69,6 +69,7 @@
 #include "xmlparse.h"
 #include "xlsxparse.h"
 #include "xpsxmlparse.h"
+#include "hashterm.h"
```

Comment:
When adding something to a list that's already in sorted order, add it in the correct place.

Reply:
I had not noticed the alphabetical order, I will fix it.